### PR TITLE
Update tabular tracker and fix doughnut variable names

### PR DESCRIPTION
**Pull request by michellewang**
Closes #75.

Also formats the tabular bagel to the format accepted by the dashboard.

### CODE REVIEW

File: nipoppy/workflow/tabular/tabular_tracker.py [hunk 1]
```diff
@@ -23,6 +25,101 @@
...
+DASH_BAGEL_ID_COLS = [COL_BIDS_ID_MANIFEST, COL_SUBJECT_MANIFEST, COL_VISIT_MANIFEST]
+DASH_BAGEL_VAR_NAME = 'assessment_name'
+DASH_BAGEL_VAR_VALUE = 'assessment_score'
+
+COL_UPDRS3 = 'NP3TOT'
+COL_AGE = 'AGE_AT_VISIT'
+
+def loading_func(df):
+    if COL_UPDRS3 in df.columns:
+        print(f'Filtering {COL_UPDRS3}')
```
Comment (nikhil153) [inline nipoppy/workflow/tabular/tabular_tracker.py:39]: not urgent, but switch to logger?

File: nipoppy/workflow/tabular/tabular_tracker.py [hunk 2]
```diff
@@ -23,6 +25,101 @@
...
+    return df_on_off
+
+def age_filter(df: pd.DataFrame):
+    def visit_sort_key(visit):
+        if visit == 'SC':
+            return -10
+        elif visit == 'BL':
+            return -5
+        else:
+            return int(''.join([c for c in visit if c.isdigit()]))
```
Comment (nikhil153) [inline nipoppy/workflow/tabular/tabular_tracker.py:80]: What are these numbers? Add some comments?

File: nipoppy/workflow/tabular/tabular_tracker.py [hunk 3]
```diff
@@ -23,6 +25,101 @@
...
+            return series
+    
+    # print(df.shape)
+    # print(df[[COL_SUBJECT_TABULAR, COL_VISIT_TABULAR]].drop_duplicates().shape)
+    df[COL_AGE] = df[COL_AGE].astype(float)
+
+    # find subjects with multiple age entries for the same visit
+    groups = df.groupby([COL_SUBJECT_TABULAR, COL_VISIT_TABULAR])[COL_AGE]
+    counts = groups.count()
+    records_with_multiple_ages = counts[counts > 1].index.unique()
```
Comment (nikhil153) [inline nipoppy/workflow/tabular/tabular_tracker.py:98]: We should add what we mean by `record` in the docs.

File: nipoppy/workflow/tabular/tabular_tracker.py [hunk 4]
```diff
@@ -23,6 +25,101 @@
...
+        # print(f'final_age: {final_age}')
+
+        df_no_duplicates.loc[record_to_fix, COL_AGE] = final_age
+
+    df_no_duplicates = df_no_duplicates.reset_index()
+    df_no_duplicates = df_no_duplicates.sort_values(by=[COL_SUBJECT_TABULAR, COL_VISIT_TABULAR], key=subject_sort_key)
+    # print(df_no_duplicates.shape)
+    # print(df_no_duplicates[[COL_SUBJECT_TABULAR, COL_VISIT_TABULAR]].drop_duplicates().shape)
+    return df_no_duplicates
+
```
Comment (nikhil153) [inline nipoppy/workflow/tabular/tabular_tracker.py:122]: We need some notes on how you are resolving the "bad_ages"

**Review by nikhil153 — APPROVED** [review]
Needs a bit more documentation on the logic flow. 
All good otherwise!

File: nipoppy/workflow/tabular/tabular_tracker.py (hunk 2 above)
Comment (michellewang) [reply nipoppy/workflow/tabular/tabular_tracker.py:81]: It's just the make sure that `SC` is before `BL` when sorting with this key. The actual number doesn't matter, just the relative ordering. Added a comment

**Pull request merged**